spec/requests/api/providers_spec.rb

```diff
@@ -438,6 +455,19 @@ def token(connection)
     end
   end
 
+  describe "custom attributes assignment" do
+    it "custom attributes assignment" do
+      api_basic_authorize collection_action_identifier(:providers, :update_custom_attributes)
+      p1 = FactoryGirl.create(:ext_management_system, :name => "provider")
+      run_post(providers_url, gen_request(:update_custom_attributes, custom_attributes_request))
+      expect(p1.custom_attributes.count).to eq(3)
+      expect(CustomAttribute.find_by_name("experation_date").serialized_value.class).to eq(Date)
```

Comment:
@alongoldboim I couldn't comment on that - but if you're testing some behavior that's not exposed through the API it may be more appropriate to write a lower-level test